src/demo/app/components/directive-button-demo.component.html

```diff
@@ -19,6 +19,8 @@ <h4><em>How it works</em></h4>
 </button>
```

Comment:
please revert all changes to this file.  This is not something that should have been included in this pull request.

Reply:
Going to comment this on the timeline, we spoke about it.